.github/workflows/integration-test.yml

```diff
@@ -69,12 +69,12 @@ jobs:
         run: make integration-test
 
       - name: Create reports
-        if: always()
+        if: ${{ failure() }}
         working-directory: ./.github/scripts
         run: ./create-reports-full.sh
 
       - name: Upload cluster logs
-        if: always()
+        if: ${{ failure() }}
```

Comment:
you don't need `{{}}` here

Reply:
yeah, I tend to use `${{ }}` to be ready to evolve the expression :) but I don't expect here to have any fancy bool expr